.flowconfig

```diff
@@ -10,6 +11,8 @@
 [include]
 
 [libs]
+./flow/interfaces/
```

Comment:
nit: we can use `flow-typed` directory and not configure `[libs]` in `.flowconfig` as `flow-typed` is dir automatically inspected by Flow.